src/main/kotlin/app/hashers/CodeLongevity.kt

```diff
@@ -186,8 +198,12 @@ class CodeLongevity(repoPath: String, tailRev: String) {
                 }
 
                 // If a file was deleted, then the new path is /dev/null.
-                val path = if (newPath != DiffEntry.DEV_NULL) newPath else oldPath
-                var lines = files.get(path)!!
+                val path = if (newPath != DiffEntry.DEV_NULL) {
```

Comment:
why did this change? the original looked more compact.

Reply:
Doesn't fit 80 limit.